packages/expo-updates/src/Updates.types.ts

```diff
@@ -3,7 +3,8 @@ import { NewManifest, BareManifest } from 'expo-manifests';
 export type Manifest = NewManifest | BareManifest;
 
 /**
- * The types of update-related events.
+ * @deprecated The types of update-related events, used with `addListener()` and `useUpdateEvents()`.
+ * These APIs are deprecated, and may be removed in a future release; `useUpdates()` should be used instead.
```

Comment:
```suggestion
 * These APIs are deprecated and will be removed in a future release. Use [`useUpdates()`](#useupdates) instead.
```

Let's add the internal link to the hook definition. Also, even though we are not sure when it will be removed (if that's the case), let's be a little authoritative here that this will be deprecated.